README.md

```diff
@@ -1,29 +1,133 @@
-# ForemanKubevirt
 
-This plugin enables managing of Kubevirt Provider as a Compute Resource in Foreman.
+# Foreman Kubevirt Plugin
+
+The ```foreman_kubevirt ``` plugin enables managing of [KubeVirt](https://kubevirt.io) as a Compute Resource in Foreman.
+
+* Website: [TheForeman.org](http://theforeman.org)
+* Wiki: [Foreman wiki](http://projects.theforeman.org/projects/foreman/wiki/About)
+* Community and support: #theforeman for general support, #theforeman-dev for development chat in [Freenode](irc.freenode.net)
+* Mailing lists:
+    * [foreman-users](https://groups.google.com/forum/?fromgroups#!forum/foreman-users)
+    * [foreman-dev](https://groups.google.com/forum/?fromgroups#!forum/foreman-dev)
+
 
 ## Installation
 
-See [How_to_Install_a_Plugin](http://projects.theforeman.org/projects/foreman/wiki/How_to_Install_a_Plugin)
-for how to install Foreman plugins
+Please see the Foreman manual for appropriate instructions:
+
+* [Foreman: How to Install a Plugin](https://theforeman.org/plugins/#2.Installation)
+
+However, since the plugin isn't yet accepted to the foreman organization, packaging and installing should be done manually.
+In order to install *foreman_kubevirt* plugin manually on Foreman, follow this steps:
+### Building the plugin
+    # git clone https://github.com/masayag/foreman_kubevirt.git
+    # cd foreman_kubevirt
+    # gem build foreman_kubevirt.gemspec # the output will be gem named foreman_kubevirt-x.y.z.gem, where x.y.z should be replaced with the actual version
+
+### Installing the plugin
+
+#### Red Hat, CentOS, Fedora, Scientific Linux (rpm)
+    # sudo -i
+    # scl enable tfm bash
+    # yum -y install gcc-c++ redhat-rpm-config gcc rubygems rh-ruby25-ruby-devel-2.5 # or a matching version according to the installed ruby
+    # gem install foreman_kubevirt-x.y.z.gem # replace x.y.z with the actual version
+
```

Comment:
the title is not so clear to me, maybe it should be something like " Add foreman_kubevirt gem to foreman"

Reply:
This section should look like this:

![Selection_999(146)](https://user-images.githubusercontent.com/316242/54185724-64f29e00-44b2-11e9-91a4-30c01146080f.png)

Do you think it requires changes ?

Reply:
"Builiding and Installing the plugin from source"
1. You have a typo, should be Building instead of Builiding
2. shouldn't it be a title for "building the plugin" and "installing the plugin" ?

Reply:
@shiramax 
![Selection_999(147)](https://user-images.githubusercontent.com/316242/54188525-8ce50000-44b8-11e9-8f61-88412208a0a0.png)